torch/fx/experimental/Partitioner.py

```diff
@@ -1,6 +1,29 @@
 from torch.fx.graph_module import GraphModule
 from torch.fx.node import Node
-from typing import Dict, List, Union
+from typing import Dict, List, Union, Set, NamedTuple, Tuple
+import torch
+from torch.fx.experimental.subgraph_creation_example import split_module
+def _get_users_mapping(fx_module: GraphModule) -> Dict[Node, List[Node]]:
```

Comment:
`users` is now kept track of inside `Node` itself, so you can use that instead: https://fburl.com/sxg2ejyz